### PR TITLE
Fix the javadoc 8 errors in doctags

### PR DESCRIPTION
**Pull request by stephenc**
@reviewbybees

### CODE REVIEW

File: hpi-archetype/src/main/resources/HelloWorldBuilder/help-useFrench.html [hunk 1]
```diff
@@ -2,5 +2,5 @@
   This HTML fragment will be injected into the configuration screen
   when the user clicks the 'help' icon. See global.jelly for how the
   form decides which page to load.
-  You can have any <tt>HTML</tt> fragment here.
+  You can have any {@code HTML} fragment here.
```
Comment (jglick) [inline hpi-archetype/src/main/resources/HelloWorldBuilder/help-useFrench.html]: Why does this need to be monospace at all?!

Comment (jglick) [inline hpi-archetype/src/main/resources/HelloWorldBuilder/help-useFrench.html]: :bug: this is not Javadoc, it is raw HTML; revert.

File: src/main/java/org/jenkinsci/maven/plugins/hpi/PropertyUtils.java [hunk 2]
```diff
@@ -28,8 +28,8 @@
 /**
  * @author <a href="mailto:kenney@neonics.com">Kenney Westerhof</a>
  * @version $Id: PropertyUtils.java 10807 2008-07-14 18:56:05Z btosabre $
- * @todo this is duplicated from the resources plugin - migrate to plexus-utils
  */
+// todo this is duplicated from the resources plugin-migrate to plexus-utils
```
Comment (jglick) [inline src/main/java/org/jenkinsci/maven/plugins/hpi/PropertyUtils.java]: I thought all-caps `TODO` was conventional.

File: src/main/java/org/jenkinsci/maven/plugins/hpi/VersionNumber.java [hunk 3]
```diff
@@ -46,16 +46,16 @@
...
- * '*' is also allowed as a component, and '*' > M for any M > 0.
+ * '*' is also allowed as a component, and '*' &gt; M for any M &gt; 0.
  *
  * <p>
  * 'SNAPSHOT' is also allowed as a component, and "N.SNAPSHOT" is interpreted as "N-1.*"
  *
  * <pre>
- * 2.0.* > 2.0.1 > 2.0.1-SNAPSHOT > 2.0.0.99 > 2.0.0 > 2.0.ea > 2.0
+ * 2.0.* &gt; 2.0.1 &gt; 2.0.1-SNAPSHOT &gt; 2.0.0.99 &gt; 2.0.0 &gt; 2.0.ea &gt; 2.0
  * </pre>
```
Comment (jglick) [inline src/main/java/org/jenkinsci/maven/plugins/hpi/VersionNumber.java]: Wrapping in `{@code … }` would probably be more readable.

File: hpi-archetype/src/main/resources/HelloWorldBuilder/help-useFrench.html (hunk 1 above)
Comment (stephenc) [inline hpi-archetype/src/main/resources/HelloWorldBuilder/help-useFrench.html]: Got a bit too zealous with search and replace

**Comment by jglick** [comment]
:bee:

**Pull request merged**